docs/content/latest/explore/multi-region-deployments/read-replicas.md

```diff
@@ -0,0 +1,281 @@
+44 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Run time (seconds): -1
+44 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Adding node: 127.0.0.1:9042
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Adding node: 127.0.0.2:9042
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Adding node: 127.0.0.3:9042
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Num reader threads: 1, num writer threads: 1
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Num unique keys to insert: 2
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Num keys to update: 1999998
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Num keys to read: 1500000
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Value size: 1024
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Restrict values to ASCII strings: false
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Perform sanity check at end of app run: false
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Table TTL (secs): -1
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Local reads: false
+45 [main] INFO com.yugabyte.sample.common.CmdLineOpts  - Read only load: false
+48 [main] INFO com.yugabyte.sample.apps.AppBase  - Creating Cassandra tables...
+92 [main] INFO com.yugabyte.sample.apps.AppBase  - Connecting with 4 clients to nodes: /127.0.0.1:9042,/127.0.0.2:9042,/127.0.0.3:9042
+1139 [main] INFO com.yugabyte.sample.apps.AppBase  - Created a Cassandra table using query: [CREATE TABLE IF NOT EXISTS CassandraKeyValue (k varchar, v blob, primary key (k));]
+6165 [Thread-1] INFO com.yugabyte.sample.common.metrics.MetricsTracker - Read: 4009.75 ops/sec (0.24 ms/op), 20137 total ops | Write: 1650.50 ops/sec (0.60 ms/op), 8260 total ops | Uptime: 5025 ms | maxWrittenKey: 1 | maxGeneratedKey: 2 | 
+11166 [Thread-1] INFO com.yugabyte.sample.common.metrics.MetricsTracker - Read: 5066.60 ops/sec (0.20 ms/op), 45479 total ops | Write: 1731.19 ops/sec (0.58 ms/op), 16918 total ops | Uptime: 10026 ms | maxWrittenKey: 1 | maxGeneratedKey: 2 | 
+```
+
+The following illustration demonstrates the read and write statistics in the primary cluster visible via [Yugabyte Platform](https://docs.yugabyte.com/latest/yugabyte-platform/):
+
+![img](/images/explore/multi-region-deployments/read-replicas2.png)
+
+As per the preceding illustration, using the default workload directs reads and writes to the tablet leader. The arguments in the `java -jar ./yb-sample-apps.jar` command explicitly restrict the number of keys to be written and read to one in order to follow the reads and writes occurring on a single tablet.
+
+The following is a modified command that enables follower reads. Specifying `--local_reads` changes the consistency level to `ONE`. The `--with_local_dc` option defines in which universe the application is at any given time. When specified, the read traffic is routed to the same region:
```

Comment:
universe -> datacenter. datacenter is a Cassandra/YCQL concept and a config parameter.

Reply:
Changed